common/djangoapps/pipeline_mako/templates/static_content.html

```diff
@@ -97,32 +99,22 @@
             -include it as the first script in this block
     </%doc>
     <%
-        from django.template import Template, Context
-        from webpack_loader.exceptions import WebpackLoaderBadStatsError
-        import json
-
         def _convert_dict_to_json(input_dict):
             output_json = "{"
             for key in input_dict:
                 output_json = "{}{}:\"{}\",".format(output_json, key, input_dict[key])
             output_json += "}"
             return output_json
 
-        body = capture(caller.body)
+        body = _convert_dict_to_json(capture(caller.body))
```

Comment:
Should this just be `json.dumps`? I don't know the history of `_convert_dict_to_json`, but it seems like an unnecessary and incomplete implementation of `json.dumps`.

Reply:
Actually, this was apparently a recent addition, so I'm just pinging the author now on why it was implemented that way.

Reply:
Ah, apparently this method was actually removed on master recently: https://github.com/edx/edx-platform/pull/16734/files#diff-1eb576102f40a534c2862d86fc3ff321L104